bundles/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -114,3 +132,29 @@ then
 end 
 
 ```
+
+dwdunwetter_severity.map
```

Comment:
Please rename these transformation files to `dwdunwetter_severity_de.map` as it contains German mappings.

Reply:
Thanks... Changed and fixed